### PR TITLE
Added paging controls at the bottom of the page

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/js/vue/components/search/ResultSet.vue [hunk 1]
```diff
@@ -42,13 +42,26 @@
...
         </section>
-        <b-pagination
-                v-model="currentPage"
-                :total-rows="rows"
-                :per-page="formValues.perPage"
-                class="bg justify-content-center"
-                style="margin-bottom: 100px;">
-        </b-pagination>
+
+        <div class="container">
```
Comment (mickel1138) [inline assets/js/vue/components/search/ResultSet.vue:46]: Maybe this should be a component?

Comment (mickel1138) [inline assets/js/vue/components/search/ResultSet.vue:46]: If you move this div in the `<section>` above, it looks better. There is a stray gray box in the front.

Comment (praneethpr) [reply assets/js/vue/components/search/ResultSet.vue:46]: I don't see the benefit of making this pagination to be a component, its more work to get events propagated from the child back to the parent, then submit the form in the grandparent component

Comment (praneethpr) [reply assets/js/vue/components/search/ResultSet.vue]: Moved it

**Pull request merged**